### ISSUE TITLE
master - product_option_form.twig

### ISSUE THREAD

**Issue by Pelham81** [opencart/opencart#6324]
For the subtract stock column where you select "yes" or "no", the value does not show up correctly after you save (even though it gets saved correctly in the db). If you select yes and save, it shows as no - and vice versa.

admin/view/template/catalog/product_option_form.twig

lines 159 through 165 show:
```
{% if product_option_value.subtract %}
   <option value="1" selected="selected">{{ text_yes }}</option>
   <option value="0">{{ text_no }}</option>
{% else %}
    <option value="1">{{ text_yes }}</option>
    <option value="0" selected="selected">{{ text_no }}</option>
{% endif %}
```
Should be changed to:
```
{% if product_option_value.subtract == '0' %}
   <option value="0" selected="selected">{{ text_no }}</option>
{% else %}
   <option value="0">{{ text_no }}</option>
{% endif %}
{% if product_option_value.subtract == '1' %}
   <option value="1" selected="selected">{{ text_yes }}</option>
{% else %}
   <option value="1">{{ text_yes }}</option>								
{% endif %}
```

And then it works as correctly. (I happen to prefer to show no first instead of yes, but that's just my preference which has nothing to do with this so fix as you wish).

**Comment by vuchastyi** [comment]
I think
```
{% if product_option_value.subtract == '0' %}
   <option value="1" selected="selected">{{ text_yes }}</option>
   <option value="0">{{ text_no }}</option>
{% else %}
    <option value="1">{{ text_yes }}</option>
    <option value="0" selected="selected">{{ text_no }}</option>
{% endif %}
```
is enough

**Comment by JesseOC** [comment]
Thanks! It is already fixed on #6359
